api/v3/Payment.php

```diff
@@ -169,6 +169,16 @@ function _civicrm_api3_payment_create_spec(&$params) {
       'type' => CRM_Utils_Type::T_INT,
       'api.aliases' => array('payment_id'),
     ),
+    'payment_type' => array(
+      'title' => 'Payment Type',
+      'type' => CRM_Utils_Type::T_STRING,
+      'description' => ts("Type of payment - 'owed' or 'refund'."),
+    ),
+    'is_email_receipt' => array(
```

Comment:
@monishdeb so the thing I've been saying is I think instead of adding these 2 parameters we should add a call above to Payment.sendconfirmation
- the reason being I'm not convinced that is_email_receipt should sent the payment notification - I think it should likely be a bool for whether completetransaction type emails go out - but we can punt that decision by using the payment.sendconfirmation for now

Reply:
Oops I missed it, removing it

Reply:
@monishdeb once you merge this https://github.com/civicrm/civicrm-core/pull/13610 we can switch over to Payment.sendconfirmation